### ISSUE TITLE
Transition tiles do not connect properly

### ISSUE THREAD

**Issue by jaythebusinessgoose** [spelunky-fyi/Playlunky#24]
When the game is launched through Playlunky, the tiles in transitions don't connect properly. Tiles that are supposed to be 2-tall connections draw only the bottom half twice. Or it could be that a bottom half of a connection tile is being used in random places instead of just with its top half.

**Comment by Malacath-92** [comment]
Do you mean any and all FLOOR_TUNNEL_* entities, including the ones that the game spawns usually.

**Comment by jaythebusinessgoose** [comment]
Yes, I should have specified that this is in Speedrun mode so all I have applied is skin mod, and it is the tunnel tiles in the vanilla transitions. They look fine if I launch vanilla.

**Comment by Malacath-92** [comment]
Do you know when this started happening? Do you know if this happens with Overlunky? Because I have to assume that some OL initialization affects this as Playlunky doesn't init any scripts or console or the like and doesn't hook anything crucial itself other than fmod at this point.

**Comment by jaythebusinessgoose** [comment]
And I'm not sure about OL, I hardly ever use it so I haven't tested it.

**Comment by Malacath-92** [comment]
> November 10 of last year
Oh god, why do we hear this just now  😭

**Comment by jaythebusinessgoose** [comment]
Ok, yep, I launched the game vanilla and didn't see any issues then launched Overlunky and it immediately started happening. Should I file a separate issue there or can this one transfer over?
